src/Sia.Data.Incident/Filters/EventFilters.cs

```diff
@@ -28,6 +30,11 @@ public override IQueryable<Event> Filter(IQueryable<Event> source)
             if (EventTypes != null && EventTypes.Length > 0) working = working.Where(ev => EventTypes.Contains(ev.EventTypeId));
             if (Occurred.HasValue) working = working.Where(ev => ev.Occurred == Occurred);
             if (EventFired.HasValue) working = working.Where(ev => ev.EventFired == EventFired);
```

Comment:
Given that we don't expect the Occurred and EventFired values to ever be matched exactly, the range-based filtering will be much more effective than this matching filtering, so it makes sense to remove the Occurred and EventFired properties and these comparisons.